.circleci/config.yml

```diff
@@ -0,0 +1,32 @@
+# Javascript Node CircleCI 2.0 configuration file
+#
+# Check https://circleci.com/docs/2.0/language-javascript/ for more details
+#
+version: 2
+jobs:
+  build:
+    docker:
+      # specify the version you desire here
+      - image: circleci/node:8-browsers
```

Comment:
any reason for using node version 8 here instead of the latest LTS?

Reply:
yep, will do that, and will update our Dockerfile as well.